apps/web/src/components/Router.tsx

```diff
@@ -796,6 +796,45 @@ const createRoute = (locale: Locales, newApplicantDashboard: boolean) =>
                       "../pages/PoolCandidates/ViewPoolCandidatePage/ViewPoolCandidatePage"
                     ),
                 },
+                {
+                  path: "talent-events",
+                  children: [
+                    {
+                      path: ":eventId/nominations/:talentNominationGroupId",
+                      children: [
+                        {
+                          lazy: () =>
+                            import(
+                              "../pages/TalentNominations/NominationGroup/Layout"
+                            ),
+                          children: [
+                            {
+                              index: true,
```

Comment:
I know this is a little different from the AC, but I like it.  Avoids the need to have a 404 and the root resource. :+1:

Reply:
Glad this went well, I was rolling the dice on this one :sweat_smile: